libnavigation-core/api/current.txt

```diff
@@ -569,52 +568,52 @@ package com.mapbox.navigation.core.telemetry.events {
-    field public static final String HAZARD = "hazard";
-    field public static final String INACCURATE_GPS = "inaccurate_gps";
     field public static final String INCORRECT_AUDIO_GUIDANCE = "incorrect_audio_guidance";
     field public static final String INCORRECT_SPEED_LIMIT = "incorrect_speed_limit";
+    field public static final String INCORRECT_VISUAL = "incorrect_visual";
     field public static final String INCORRECT_VISUAL_GUIDANCE = "incorrect_visual_guidance";
     field public static final com.mapbox.navigation.core.telemetry.events.FeedbackEvent INSTANCE;
     field public static final String INSTRUCTION_MISSING = "instruction_missing";
     field public static final String INSTRUCTION_UNNECESSARY = "instruction_unnecessary";
     field public static final String LANE_GUIDANCE_INCORRECT = "lane_guidance_incorrect";
     field public static final String MANEUVER_INCORRECT = "maneuver_incorrect";
-    field public static final String MISSING_EXIT = "missing_exit";
     field public static final String MISSING_ROAD = "missing_road";
-    field public static final String NOT_ALLOWED = "not_allowed";
+    field public static final String OTHER_ISSUE = "other_issue";
     field public static final String POSITIONING_ISSUE = "positioning_issue";
     field public static final String PRONUNCIATION_INCORRECT = "pronunciation_incorrect";
     field public static final String REROUTE = "reroute";
     field public static final String ROAD_CLOSED = "road_closed";
-    field public static final String ROAD_IS_MISSING_FROM_MAP = "road_is_missing_from_map";
-    field public static final String ROAD_KNOW_BY_DIFFERENT_NAME = "road_know_by_different_name";
+    field public static final String ROAD_ISSUE = "road_issue";
     field public static final String ROAD_NAME_REPEATED = "road_name_repeated";
     field public static final String ROUTED_DOWN_A_ONE_WAY = "routed_down_a_one_way";
     field public static final String ROUTE_HAD_ROADS_TOO_NARROW_TO_PASS = "route_had_roads_too_narrow_to_pass";
     field public static final String ROUTE_INCLUDED_MISSING_ROADS = "route_included_missing_roads";
-    field public static final String ROUTE_NOT_DRIVE_ABLE = "route_not_drive_able";
+    field public static final String ROUTE_NOT_ALLOWED = "route_not_allowed";
+    field public static final String ROUTE_NOT_DRIVE_ABLE = "route_not_driveable";
     field public static final String ROUTE_NOT_PREFERRED = "route_not_preferred";
     field public static final String ROUTING_ERROR = "routing_error";
     field public static final String STREET_NAME_INCORRECT = "street_name_incorrect";
     field public static final String STREET_PERMANENTLY_BLOCKED_OFF = "street_permanently_blocked_off";
-    field public static final String TURN_AT_INTERSECTION_WAS_UNPROTECTED = "turn_at_intersection_was_unprotected";
+    field public static final String STREET_TEMPORARY_BLOCKED_OFF = "street_temporary_blocked_off";
```

Comment:
"temporarily" should be used instead of "temporary", lets change both the constant name and value

Reply:
it was already used with 1.x and as far as I see iOS uses the same conts https://github.com/mapbox/mapbox-navigation-ios/blob/main/Sources/MapboxCoreNavigation/PassiveNavigationFeedbackType.swift#L53

Reply:
why would you change that?

Reply:
On iOS it was merged yesterday, but not we've caught this typo and there is a quick fix PR before the release: https://github.com/mapbox/mapbox-navigation-ios/pull/3337
I don't get how this constant was already used in 1.x if this PR is one day old.🤔

Reply:
right, mixed up it with `*permanently*`. @Zayankovsky could you address the issue with this for Android?

Reply:
Sure thing.